contracts/consensus-gateway/ConsensusCogatewayI.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @title An interface for consensus cogateway contract.
+ */
+interface ConsensusCogatewayI {
+
+    /* External functions */
+
+    /**
+     * @notice Get the open kernel hash for a given kernel height.
```

Comment:
I am a little bit confused in between function name and documentation.
I would rather assume to see:
  `getOpenKernelHash()` that returns open kernel hash
or
  `getKernelHash(uint256 _kernelHeight)` that returns kernel hash at the specified height.